tests/test_gas_source_sink.py

```diff
@@ -8,9 +8,15 @@
 
 
 class TestMILPGasSourceSink(TestCase):
-    """Unit tests for the MILP test case of a source, a pipe, a sink"""
-
     def test_source_sink(self):
+        """
+        Test for  case of a source, a pipe, a sink.
```

Comment:
- I do not understand the current sentence
- Suggested wording: "Test case for a network consisting out of a source, pipes and a sink"

Reply:
done